src/pages/Appointments/AppointmentsPage.tsx

```diff
@@ -84,17 +85,31 @@ import {
 } from "@/types/scheduling/schedule";
 import scheduleApis from "@/types/scheduling/scheduleApi";
 
+interface DateRangeDisplayProps {
+  dateFrom: string | null;
+  dateTo: string | null;
+}
+
 interface QueryParams {
   practitioner: string | null;
   slot: string | null;
+  page: number | null;
+  status: string | null;
   date_from: string | null;
   date_to: string | null;
   search: string | null;
 }
-
-interface DateRangeDisplayProps {
-  dateFrom: string | null;
-  dateTo: string | null;
+interface FilterParamsInterface {
+  qParams: any;
+  resultsPerPage: number;
+  updateQuery: (filter: FilterState) => void;
+  Pagination: ({
+    totalCount,
+    noMargin,
+  }: {
+    totalCount: number;
+    noMargin?: boolean;
+  }) => JSX.Element;
 }
```

Comment:
i don't see any reason why this is required; see how pagination is done in other places in care; we don't do this